src/foundry/client/config.d.mts

```diff
@@ -356,23 +356,27 @@ declare global {
        * }
        * ```
        */
-      termTypes: Record<string, ConstructorOf<RollTerm>>;
+      termTypes: Record<string, ConstructorOf<foundry.dice.terms.RollTerm>>;
 
-      /**
-       * Configured roll terms and the classes they map to.
-       */
+      /** Configured roll terms and the classes they map to. */
       terms: {
-        c: typeof Coin;
-        d: typeof Die;
-        f: typeof FateDie;
-      } & Record<string, ConstructorOf<DiceTerm>>;
+        c: typeof foundry.dice.terms.Coin;
+        d: typeof foundry.dice.terms.Die;
+        f: typeof foundry.dice.terms.FateDie;
+      } & Record<string, ConstructorOf<foundry.dice.terms.DiceTerm>>;
 
       /**
        * A function used to provide random uniform values.
        * @defaultValue `MersenneTwister.random`
        */
       randomUniform: () => number;
-    } & Record<string, ConstructorOf<Roll>>; // Common pattern
+
+      /** A parser implementation for parsing Roll expressions. */
+      parser: typeof foundry.dice.RollParser;
+
+      /** A collection of custom functions that can be included in roll expressions.*/
+      functions: Record<string, RollFunction>;
+    } & Record<string, ConstructorOf<foundry.dice.Roll>>; // Common pattern
```

Comment:
We may consider making this an interface. No action necessary at the moment though.

Reply:
The Record is something I added to resolve a separate ticket a bit ago; interface wouldn't be the worst idea.